hash.go

```diff
@@ -382,3 +384,45 @@ func (db *RoseDB) HIncrBy(key, field []byte, incr int64) (int64, error) {
 	}
 	return valInt64, nil
 }
+
+// HRandfield return a random field from the hash value stored at key, when called with just
+// the key argument. If the provided count argument is positive, return an array of distinct
+// fields. If called with a negative count, the behavior changes and the command is allowed
+// to return the same field multiple times.
+func (db *RoseDB) HRandfield(key []byte, count int) ([][]byte, error) {
```

Comment:
`HRANDFIELD` command accepts `WITHVALUES` option. I think you accept it like this:

```go
func (db *RoseDB) HRandfield(key []byte, count int, withValues ...bool) ([][]byte, error) {
```
Or, maybe you don't use `...` in here. It might not be appropriate in this case. Also, I suggest defining a custom type for these options in general cases. It can be actually `bool` type but with a custome name. Firstly, you may need to check the codebase and if there are similar usage, you may define a type like this:

```go
type WithValues bool
```
So, I think it makes it more readable.

Reply:
Thanks for pointing that out.
I would refer you to the following.

https://github.com/flower-corp/rosedb/pull/178#discussion_r912365996

It's not that I don't want to do it.
It's just that I haven't dealt with optional arguments before and I'm not sure if I should start implementing them haphazardly.

What do you think?

Reply:
I think we can start to implement in this PR. Maybe @roseduan has some thoughts.

Reply:
Also, to be more clear, I suggested defining a new type for `WithValues`. Maybe you can create constant variables. I am not sure which one could be more appropriate. Maybe you can add an example then.

Reply:
Thanks for the reply.
I will proceed as is in the direction of implementing this.